js/index.js

```diff
@@ -1 +1,65 @@
-// Your code goes here
+// functions
+function beepBeep() {
+  return alert('beep beep');
+}
+
+function bgPink() {
+  return event.target.style.backgroundColor = 'pink';
+}
+
+function bgDodgerBlue() {
+  return event.target.style.backgroundColor = 'dodgerblue';
+}
+
+function bgSeaGreen() {
+  return event.target.style.backgroundColor = 'seagreen';
+}
+
+function textOrange() {
+  return event.target.style.color = 'orange';
+}
+
+function resize() {
+  let width = 100;
+  return event.target.style.width = `${width-50}%`;
+}
+
+function preDefault() {
```

Comment:
Interesting choice to define all of your functions at the top. It makes your code more modular/reusable/organized. That's a good habit to create for yourself. 👍